Client/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.h

```diff
@@ -29,6 +29,7 @@ class CLuaPlayerDefs : public CLuaDefs
     LUA_DECLARE(GetPlayerTeam);
     LUA_DECLARE(GetPlayerMoney);
     LUA_DECLARE(GetPlayerWantedLevel);
+    static const std::uint8_t GetPlayerScriptDebugLevel() noexcept;
```

Comment:
const here is unnecessary and doesn't make much sense

Reply:
Thanks